requirements/edx/kernel.in

```diff
@@ -113,7 +113,8 @@ olxcleaner
 openedx-atlas                       # CLI tool to manage translations
 openedx-calc                        # Library supporting mathematical calculations for Open edX
 openedx-django-require
-openedx-events>=8.3.0               # Open edX Events from Hooks Extension Framework (OEP-50)
+# openedx-events 8.6.0 introduces publishing via configuration. Ticket to unpin: https://github.com/edx/edx-arch-experiments/issues/381
+openedx-events<8.6.0               # Open edX Events from Hooks Extension Framework (OEP-50)
```

Comment:
I think only > constraints belong here, and this belongs in constraints.txt, right?